src/common/common.js

```diff
@@ -51,6 +50,22 @@ const jsdoc = new RegExp(/@\w+\b/);
  */
 const jsdocText = new RegExp(/@\w+\b.*/g);
 
+const isClassMethod = '(^\\s*(static|async){0,1}\\s*(?!if\\s)\\w*\\s*\\()'; // prevent matching if statement "if (!x) x = {};"`
```

Comment:
Although unlikely, this RegEx would not allow static async function.

Reply:
Perhaps we can take opportunity and align naming of RegEx, changing this to `aClassMethod`?

Reply:
I think we could probably just change it to  `(static|async|static async)` there and it should be fine now that I look at it again.